Assets/Scripts/InventoryScripts/ManagerScript.cs

```diff
@@ -5,80 +5,59 @@
 
 public class ManagerScript : MonoBehaviour {
 
-    public GameObject invPanel, journalInvPanel, itemInfo;    //reference to a GameObject
-    public InventoryScript inventoryScript;                     //reference to InventoryScript
-    public JournalInventoryScript journalScript;
-    [SerializeField] Text nameOfItem, descriptionOfItem;        //Serialize Text fields for information panel
+    [SerializeField] private GameObject _invPanel, _journalInvPanel, _itemInfo;    //reference to a GameObject
+    [SerializeField] private InventoryScript _inventoryScript;                     //reference to InventoryScript
+    [SerializeField] private JournalInventoryScript _journalScript;
+    [SerializeField] private Text _nameOfItem, _descriptionOfItem;        //Serialize Text fields for information panel
     [HideInInspector]
-    public bool active = false;    // bool to check if inventory is open or closed
+    public bool active;    // bool to check if inventory is open or closed
```

Comment:
shouldn't this just be a property if its hidden in the inpsector?

Reply:
Another thing that is used by other types, but isn't needed to be seen in the inspector. Advice?

Reply:
Make it a property then.

Reply:
Example:
```public bool Active { get; private set; }```